L1Trigger/L1THGCal/interface/HGCalCoarseTriggerCellMapping.h

```diff
@@ -34,6 +34,7 @@ class HGCalCoarseTriggerCellMapping {
   static constexpr int kSplit_v8_VeryFine_ = 0x3e;
   static constexpr int kSplit_v8_Individual_ = 0x3f;
   static constexpr int kNLayers_ = 4;
+  static constexpr int kNHGCalLayers_ = 52;
```

Comment:
This value is not true for >=V10 geometries. This constant is not really containing the real number of layers but the number of layers used to fill the size vector. So it might be better to rename it to e.g. `kNHGCalLayersMax_`.
Also it is confusing to see `kNLayers_ = 4` just above; this parameter should also be renamed to something more meaningfull.